source/GameData.cpp

```diff
@@ -932,11 +932,19 @@ void GameData::LoadSources(TaskQueue &queue)
 	for(const auto &path : globalPlugins)
 		if(Plugins::IsPlugin(path))
 			LoadPlugin(queue, path);
+	globalPlugins = Files::List(Files::GlobalPlugins());
```

Comment:
This order means that zipped plugins get loaded after the non-zipped plugins. (I don't know if that is a problem, it is a change in behavior though.)

Reply:
Yes, I suppose they do... I'll have to check whether they override the regular plugins, or get skipped if they are present in both forms.

Reply:
I checked, and the plugins that are loaded earlier are preserved (so zipped plugins don't override regular ones with the same name).

Reply:
```suggestion
	// Load unzipped plugins first to give them precedence, then load the zipped plugins.
	globalPlugins = Files::List(Files::GlobalPlugins());
```